src/main/java/it/gov/pagopa/payment/controller/payment/IdPayCodePaymentController.java

```diff
@@ -1,16 +1,18 @@
 package it.gov.pagopa.payment.controller.payment;
 
 import it.gov.pagopa.payment.dto.AuthPaymentDTO;
+import it.gov.pagopa.payment.dto.idpaycode.RelateUserRequest;
+import it.gov.pagopa.payment.dto.idpaycode.RelateUserResponse;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
 @RequestMapping("/idpay/payment/idpay-code")
 public interface IdPayCodePaymentController {
 
-    @PutMapping("/{trxId}/relate-user") //TODO after refactor path
+    @PutMapping("/{transactionId}/minint/relate-user")
```

Comment:
Our BE is not interested on the source of the request: it will be the APIm to receive and authenticate it
```suggestion
    @PutMapping("/{transactionId}/relate-user")
```